loadgen/prepare-loadgen.js

```diff
@@ -22,8 +22,9 @@ const key = 'loadgenKit';
  *   'wallet' |
  *   'zoe' |
  * never} UsedHomeCaps
+ * @param { Partial<Record<string, string>>} env
  */
-export async function prepareLoadgen(home, deployPowers) {
+export async function prepareLoadgen(home, deployPowers, env) {
```

Comment:
An argument could be made that `process.env` is safely ambient. We use `process.env` directly in SES to receive configuration from the start compartment, and the notion is to shim `process.env` in non-Node.js environments if necessary. But, you’ve chosen the path of least controversy.